src/main/java/com/redhat/vertx/pipeline/Section.java

```diff
@@ -17,13 +17,13 @@
 @MetaInfServices(Step.class)
 public class Section implements Step {
     private static final Logger logger = Logger.getLogger(Section.class.getName());
-    private static List<String> RESERVED_WORDS = Arrays.asList("name", "vars", "register", "steps", "class", "timeout", "concurrent");
+    private static List<String> RESERVED_WORDS =
+            Arrays.asList("name", "register", "steps", "timeout", "concurrent", "return", "when");
```

Comment:
Are supporting both `register` and `return`?

Reply:
Okay, I understand supporting both, but when would you use return over register?